easybuild/easyconfigs/a/Anaconda/anaconda-2.4.0.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Authors::   Jillian Rowe <jillian.e.rowe@gmail.com>
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_anaconda'
+
+name = "anaconda"
+version = "2-4.0.0"
+variant = "Linux-x86_64"
+
+homepage = "https://www.continuum.io/"
+description = """Built to complement the rich, open source Python community,
+the Anaconda platform provides an enterprise-ready data analytics platform 
+that empowers companies to adopt a modern open data science analytics architecture.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+anaconda_version = 2
```

Comment:
This doesn't match the EB way of doing things... Today, if you want to install a different version of Anaconda, you need to compose a different easyconfig file and adjust the `version` spec.